components/brave_wallet_ui/components/extension/sign-panel/index.tsx

```diff
@@ -57,7 +58,11 @@ enum SignDataSteps {
   SignData = 1
 }
 
-function SignPanel (props: Props) {
+const onClickLearnMore = () => {
+  window.open('https://support.brave.com/hc/en-us/articles/4409513799693', '_blank')
```

Comment:
I see that this line wasn't changed in this PR, but I believe we need to use `chrome.tabs.create` here.

Reply:
This link doesn't seem to work for me. Do you know the article that this is supposed to reference?